apps/accounts/api/automations/check_account.py

```diff
@@ -80,7 +82,8 @@ def adhoc(self, request, *args, **kwargs):
 class AccountRiskViewSet(OrgBulkModelViewSet):
     model = AccountRisk
     search_fields = ("username", "asset")
-    filterset_fields = ("risk", "status", "asset")
+    filterset_fields = ("risk", "status", "asset_id")
+    extra_filter_backends = [NodeFilterBackend]
     serializer_classes = {
         "default": serializers.AccountRiskSerializer,
         "assets": serializers.AssetRiskSerializer,
```

Comment:
There are no known irregularities or potential issues, but if you want to optimize this version of the code, considering using a single line for constructing `SearchField` and adding `model' argument inside all custom filter set field:

```python
FiltersetClass.from_queryset(AccountRisk.objects.using("main").filter())
FilterSet.queryset = FiltersetClass.from_queryset(Model.filter(using="main"))
```
This way it is easier to work with objects directly (in django models). The rest of the code seems fine.

Let me know if there's more I can assist you on!